Description: don't let LXC_PATH= line end in failure
 Otherwise if called from dash with set -e, dash will exit.
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/lxc/+bug/1166870
Forwarded: yes

Index: lxc-0.9.0/src/lxc/lxc.functions.in
===================================================================
--- lxc-0.9.0.orig/src/lxc/lxc.functions.in	2013-04-05 04:45:22.000000000 -0500
+++ lxc-0.9.0/src/lxc/lxc.functions.in	2013-04-09 12:33:20.759554069 -0500
@@ -26,7 +26,7 @@ templatedir=@LXCTEMPLATEDIR@
 lxcinitdir=@LXCINITDIR@
 
 get_default_lxcpath() {
-	LXC_PATH=$(grep -v "^#" "$globalconf" 2>/dev/null | grep "[ \t]*lxcpath[ \t]*=")
+	LXC_PATH=$(grep -v "^#" "$globalconf" 2>/dev/null | grep "[ \t]*lxcpath[ \t]*=") || true
 	if [ -n "$LXC_PATH" ]; then
 		echo $LXC_PATH | awk -F= '{ print $2 }'
 	else
